substrate/frame/support/procedural/src/pallet/expand/pallet_struct.rs

```diff
@@ -203,9 +203,9 @@ pub fn expand_pallet_struct(def: &mut Def) -> proc_macro2::TokenStream {
 			for #pallet_ident<#type_use_gen>
 			#config_where_clause
 		{
-			type CurrentStorageVersion = #current_storage_version_ty;
+			type InCodeStorageVersion = #in_code_storage_version_ty;
 
-			fn current_storage_version() -> Self::CurrentStorageVersion {
+			fn in_code_storage_version() -> Self::InCodeStorageVersion {
```

Comment:
We should probably keep both old versions in as deprecated for downstream devs to not directly break their code.  
Although it is just a Search+Replace. Can you then please also mention this in the MR description?

Reply:
Yes, am planning to write up a PRDoc for this with instructions for migrating.

Am hoping it's a small enough change we don't need to keep the old API around, and PRDoc makes it easy enough to migrate.. Need the new stable/unstable branches :P

Reply:
Added a PRDoc and left an the old method in there (deprecated) as an alias.